src/yunohost/tools.py

```diff
@@ -852,26 +868,36 @@ def tools_migrations_migrate(target=None, skip=False, auto=False, accept_disclai
 
     # effectively run selected migrations
     for migration in migrations:
+
+        # Start register change on system
+        uo= UnitOperation('tools_migrations_migrate_' + mode)
+        uo.start()
+
         if not skip:
 
             logger.warn(m18n.n('migrations_show_currently_running_migration',
                                number=migration.number, name=migration.name))
 
             try:
                 if mode == "forward":
-                    migration.migrate()
+                    m = migration["module"].MyMigration()
+                    m.uo = uo
+                    m.migrate()
                 elif mode == "backward":
-                    migration.backward()
+                    m = migration["module"].MyMigration()
+                    m.uo = uo
+                    m.backward()
```

Comment:
Can you really avoid using one letter variable name plz? Especially when it wasn't one and a perfectly fine one before :/